### PR TITLE
Add get_job_posting 

### PR DESCRIPTION
**Pull request by aeciorc**
This method returns job posting details. The job_urn_id can be found in the "jobPosting" fields in the response of a linkedin.search for job postings,e.g:
```
      "jobPosting": "urn:li:fs_normalized_jobPosting:1754968923",
```

### CODE REVIEW

**Comment by tomquirk** [comment]
@aeciorc LGTM! However, would you be able to add a test for this? Use the other ones as examples 😄

**Comment by tomquirk** [comment]
@aeciorc wanted to double check that you were still interested in going ahead with this PR! Let me know if you need help with the tests.

**Comment by aeciorc** [comment]
Hey @tomquirk, totally forgot about this PR, my bad. I'm still interested, I'll push some tests this weekend.

File: DOCS.md [hunk 1]
```diff
@@ -217,6 +218,29 @@ linkedin = Linkedin(credentials['username'], credentials['password'])
 company = linkedin.get_company('linkedin')
 ```
 
+---
+<a name="get_job_posting"></a>
```
Comment (tomquirk) [inline DOCS.md:222]: Could we move this to a docstring? The project now uses Sphinx for auto doc generation. Have a look at other functions for examples and ping me if you need help :)

**Review by tomquirk — CHANGES_REQUESTED** [review]

**Comment by tomquirk** [comment]
Bump!

**Comment by abinpaul1** [comment]
Added in https://github.com/tomquirk/linkedin-api/pull/234

**Pull request closed without merging**